src/Core/include/OpenLoco/Core/EnumFlags.hpp

```diff
@@ -40,5 +40,9 @@ namespace OpenLoco::Core
     inline constexpr ENM operator~(const ENM a) noexcept                                                                    \
     {                                                                                                                       \
         return static_cast<ENM>(~static_cast<std::underlying_type_t<ENM>>(a));                                              \
+    }                                                                                                                       \
+    inline constexpr bool operator&&(const ENM a, const ENM b) noexcept                                                     \
```

Comment:
What is this? The operation performed is also just `A & B` and not `A && B`, we have operator& implemented, see line 31.

Reply:
it's `A & B` cast to a bool. I got tired of writing `(A & B) != ENM:none`, but if this is truly undesirable I will keep doing that.

Reply:
I think this is rather confusing, I didn't even notice the return type. It's better to have utility functions in whatever stores the flags, I was also thinking about maybe adding FlagHolder<EnumType> which will always provide has/get/set etc. Will have to look at this in the near future but for now its best to create those utility functions